lib/ip_record.js

```diff
@@ -3,37 +3,96 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // Keep track of events related to just IP addresses
-module.exports = function (BLOCK_INTERVAL_MS, now) {
+module.exports = function (BLOCK_INTERVAL_MS, RATE_LIMIT_INTERVAL_MS, MAX_ACCOUNT_STATUS_CHECK, now) {
 
   now = now || Date.now
 
+  var ACCOUNT_STATUS_ACTION = {
+    requestAccountStatus       : true
```

Comment:
naming nit: none of the other actions have a prefix like "request" on them, something like "accountStatus" or "accountStatusCheck" would be more consistent with existing actions like "passwordForgotSendCode" and "accountCreate".